vassal-doc/src/main/readme-referencemanual/ReferenceManual/MarkMoved.adoc

```diff
@@ -33,6 +33,8 @@ SEE ALSO: <<MovementTrail.adoc#top,Movement Trails>>)
 
 *Key command:*:: <<NamedKeyCommand.adoc#top,Keystroke or Named Command>> to clear the piece's move history (setting _Moved_ property back to _false_ and clearing any <<MovementTrail.adoc#top,Movement Trails>>).
 
+*Ignore moves which don't change either Location Name or Mat:*:: If checked, then don't the piece as moved if neither its Location Name nor its mat changed as a result of its move. This can be used to ignore insignificant moves (e.g. a couple pixels in an area movement game).
```

Comment:
If checked, then don't *mark* the piece as moved....

Reply:
Thx!